source/Government.cpp

```diff
@@ -148,8 +188,34 @@ void Government::Load(const DataNode &node)
 			else if(key == "atrocities")
 				atrocities.clear();
 			else
-				child.PrintTrace("Cannot \"remove\" a specific value from the given key:");
+				child.PrintTrace("Cannot \"remove\" the given key:");
+
+			// If not in "overwrite" mode, move on to the next node.
+			if(overwriteAll)
+				shouldOverwrite.erase(key);
+			else
+				continue;
+		}
+
+		if(key == "raid")
+		{
+			const Fleet *fleet = GameData::Fleets().Get(child.Token(valueIndex));
+			if(remove)
+			{
+				for(auto it = raidFleets.begin(); it != raidFleets.end(); )
+					if(it->GetFleet() == fleet)
+						it = raidFleets.erase(it);
+					else
+						++it;
+			}
+			else
+				raidFleets.emplace_back(RaidFleet(fleet,
+					(child.Size() > (valueIndex + 1) ? child.Value(valueIndex + 1) : 2.),
+					child.Size() > (valueIndex + 2) ? child.Value(valueIndex + 2) : 0.));
```

Comment:
You don't need to construct before emplacing into a vector. `emplace_back` does that for you.
```suggestion
				raidFleets.emplace_back(fleet,
					child.Size() > (valueIndex + 1) ? child.Value(valueIndex + 1) : 2.,
					child.Size() > (valueIndex + 2) ? child.Value(valueIndex + 2) : 0.);
```